src/splitseq_combine.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+#Run once for each sample to gather sequences from multiple sequencing runs that belong to that sample.  Sample name must be exactly the same as in each sequencing run.
+
+#USAGE: ./splitseq_combine.sh run1 v1 reference_directory "splitseq_all_out_directories" "sample_name A1:B6"
+
+all_args=("$@")
+
+run_id=$1
+chemistry=$2  #v1 or v2
+ssrefdir=$3
+libdirs="$4"
+sample_wells=("${all_args[@]:4}")
+
+splitseq_venv="/Genomics/grid/users/rleach/local/splitseq"
+splitseq_runs="${splitseq_venv}/splitseq_runs"
+
+cd "${splitseq_venv}"
+
+mkdir "${splitseq_runs}"
+
+set -eu
+
+. /usr/share/Modules/init/sh
+
+module load STAR/2.7.3a
+module load samtools/1.10
+module load python/3.6.4
+
+mkdir "${splitseq_runs}/${run_id}"
+
+echo ./bin/python ./bin/split-seq combine \
```

Comment:
Don't specify paths to python or split-seq. Let the user's environment define that. Unless of course, split-seq doesn't end up in the path after a `pip install` I'm just not sure how that works, perhaps I need to start trying this out. Is there a simple toy example I can run somewhere on `gen-comp2`? Perhaps in `Genomics/biocomp`? Might be helpful so I can make more cogent and relevant comments. :wink:

Reply:
The `echo` line isn't required if [`set -x`](https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/#set-x) is used, since all commands are printed before executing.